tasks/tear-up.yml

```diff
@@ -4,13 +4,27 @@
   apt:
     update_cache: "yes"
 
-- name: "Install necessary packages"
+- name: "remove broken python-pip (for ubuntu 14.04)"
```

Comment:
If this only applies to Ubuntu 14.04, should we add a `when` keyword to the task or is it not worth at the moment because officially we're still not supporting 16.04?